modules/subscriptionService/src/main/scala/routes/Routes.scala

```diff
@@ -0,0 +1,41 @@
+package routes
+
+import cats.effect.Sync
+import mockedSubscriptionResponse.MockedResponse
+import org.http4s.HttpRoutes
+import org.http4s.dsl.Http4sDsl
+import org.http4s.circe._
+import utils.TypeDecoders._
+import utils.TypeEncoders._
+import utils.Types.PostSubscriptions
+import cats.implicits._
+
+object Routes {
+
+  def subscription[F[_]: Sync]: HttpRoutes[F] = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
+    HttpRoutes.of[F] { case GET -> Root =>
+      Ok("subscription service")
+    }
+  }
+
+  def getUserSubscription[F[_]: Sync]: HttpRoutes[F] = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
+    HttpRoutes.of[F] { case GET -> subscription / user =>
+      Ok(MockedResponse.mockedGetUserSubscriptionResponse)
+    }
+  }
+
+  def postUserSubscription[F[_]: Sync]: HttpRoutes[F] = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
+
+    HttpRoutes.of[F] { case req @ POST -> subscription / user =>
+      req.decode[PostSubscriptions] { message: PostSubscriptions =>
+        Ok(message)
+      }
+    }
+  }
```

Comment:
What I usually do is group routes by domain. In this case, all these would be defined in a single method called "subscription" for example

```suggestion
  def subscription[F[_]: Sync]: HttpRoutes[F] = {
    val dsl = new Http4sDsl[F] {}
    import dsl._
    HttpRoutes.of[F] { 
      case GET -> Root =>
        Ok("subscription service")
      case GET -> subscription / user =>
        Ok(MockedResponse.mockedGetUserSubscriptionResponse)
      case req @ POST -> subscription / user =>
        req.decode[PostSubscriptions] { message: PostSubscriptions =>
          Ok(message)
        }
    }
  }
```